commons/src/main/java/com/powsybl/commons/config/ModuleConfigUtil.java

```diff
@@ -32,6 +29,13 @@ private ModuleConfigUtil() {
         return getOptionalProperty(moduleConfig, names, moduleConfig::getOptionalStringListProperty);
     }
 
+    public static Optional<Double> getOptionalDoubleProperty(ModuleConfig moduleConfig, List<String> names) {
```

Comment:
It's better to return an OptionalDouble instead of Optional<Double>

Reply:
I don't think I can here, because of the way getOptionalProperty is defined: it returns <T> Optional<T>